core/src/main/java/org/radargun/stages/test/legacy/LegacyStressor.java

```diff
@@ -94,8 +96,13 @@ private void runInternal() {
             Operation operation = operationSelector.next(random);
             try {
                logic.run(operation);
+               if (delayBetweenRequests > 0)
```

Comment:
b) is correct:) This is supposed to simulate the think time of the user/client that is sending requests to the server. In this case only latency makes sense, not throughput because the throughput is mostly given by the delay between requests. The name is "delayBetweenRequests" because there's already the same parameter in background.Stressor. So they're called same to prevent confusion.